### PR TITLE
Fix release app to GitHub with Circle CI

### LINKED ISSUE #45: Generate automatic release
Generate automatic release from git metadata

https://github.com/conventional-changelog/conventional-github-releaser

### PR DESCRIPTION
**Pull request by hectorerb**
Hi, @ajsb85

This PR adds these changes:

- Rename documentation folder
- Fix generate release with conventional-github-releaser
- Upload ipa file to GitHub release
- fix changelog header

close: https://github.com/flyve-mdm/flyve-mdm-press-kit/issues/29
close: https://github.com/flyve-mdm/flyve-mdm-press-kit/issues/28
close: https://github.com/flyve-mdm/flyve-mdm-ios-inventory-agent/issues/45

### CODE REVIEW

File: ci/scripts/deploy_master.sh [hunk 1]
```diff
@@ -45,15 +43,29 @@ if [[ $GH_COMMIT_MESSAGE != *"ci(release): generate CHANGELOG.md for version"* &
...
     # Update CHANGELOG.md on gh-pages
     git branch -D gh-pages
     git fetch origin gh-pages
     git checkout gh-pages
     git checkout $CIRCLE_BRANCH CHANGELOG.md
 
-    # Create header content
-    HEADER="---\nlayout: modal\ntitle: changelog\n---\n"
+    # Create header content to CHANGELOG.md
+    HEADER="---"$'\r'"layout: modal"$'\r'"title: changelog"$'\r'"---"$'\r\r'
```
Comment (ajsb85) [inline ci/scripts/deploy_master.sh]: Are you sure its works?

File: fastlane/Fastfile [hunk 2]
```diff
@@ -80,6 +80,37 @@ platform :ios do
...
   end
 
+  desc "builds and packages iOS apps"
+  lane :archive do
+    # Fetch provisioning profile
+    # sigh
+    sigh(
+      adhoc: false, 
+      username: "hrondon@teclib.com",
+      team_id: "H7XJV96LX2",
```
Comment (rafaelje) [inline fastlane/Fastfile]: Could you put this team_id and cert_id like a secret variable

**Review by rafaelje — DISMISSED** [review]
Reviewed 👍

File: circle.yml [hunk 3]
```diff
@@ -8,8 +8,8 @@ machine:
     SCRIPT_PATH: ./ci/scripts
     CERTIFICATES_PATH: ./ci/cert
     PROFILE_PATH: ./ci/profile
-    GH_USER: flyve-mdm-bot
-    GH_REPO_SLUG: flyve-mdm/flyve-mdm-ios-inventory-agent
```
Comment (ajsb85) [inline circle.yml]: Why you dont use `CIRCLE_PROJECT_USERNAME` and `CIRCLE_PROJECT_REPONAME`?

**Comment by ajsb85** [comment]
Hi, @hectorerb 

Rafael and me written some comments in your PR.

File: ci/scripts/deploy_master.sh (hunk 1 above)
Comment (hectorerb) [reply ci/scripts/deploy_master.sh]: yes, it was generated with this script
https://github.com/flyve-mdm/flyve-mdm-ios-inventory-agent/blob/gh-pages/CHANGELOG.md

File: fastlane/Fastfile (hunk 2 above)
Comment (hectorerb) [reply fastlane/Fastfile]: yes, that's possible

File: ci/scripts/deploy_master.sh (hunk 1 above)
Comment (ajsb85) [reply ci/scripts/deploy_master.sh]: I think `\r\n` is most common
```
\r (Carriage Return)
\r\n (End Of Line) 
\n (Line Feed)
```

File: circle.yml (hunk 3 above)
Comment (hectorerb) [reply circle.yml]: no problem, I can do it

File: ci/scripts/deploy_master.sh (hunk 1 above)
Comment (hectorerb) [reply ci/scripts/deploy_master.sh]: I'll use `\r\n`

File: circle.yml (hunk 3 above)
Comment (hectorerb) [reply circle.yml]: changed!

File: fastlane/Fastfile (hunk 2 above)
Comment (hectorerb) [reply fastlane/Fastfile]: changed

File: ci/scripts/deploy_master.sh (hunk 1 above)
Comment (hectorerb) [reply ci/scripts/deploy_master.sh]: changed

**Comment by hectorerb** [comment]
Hi, @ajsb85 

Can you check this PR?

**Review by ajsb85 — APPROVED** [review]
Reviwed

**Pull request merged**